WebViewControl/AssemblyCache.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace WebViewControl {
+
+    internal class AssemblyCache {
+
+        private static object SyncRoot { get; } = new object();
```

Comment:
readonly ... not static

Reply:
the modifier readonly is not valid for this item. I've simply removed static. In WebView.cs it's just private object